Repositories/Repositories/LevelMetadataRepository.cs

```diff
@@ -0,0 +1,63 @@
+using Npgsql;
+using Dapper;
+
+using ShmupCreator.Repositories.Models;
+
+namespace ShmupCreator.Repositories;
+
+public interface ILevelMetadataRepository
+{
+    Task Delete(LevelMetadata levelMetadata);
+    Task<IEnumerable<LevelMetadata>> GetAll();
+    Task<LevelMetadata> Insert(LevelMetadata levelMetadata);
+    Task<LevelMetadata> Update(LevelMetadata levelMetadata);
+}
+
+public class LevelMetadataRepository : ILevelMetadataRepository
+{
+    private const string connString = "Server=localhost;Port=5432;User Id=postgres;Password=FFtest4;Database=mydb;";
+
+    private async Task<LevelMetadata> GetLevelMetadataById(NpgsqlConnection con, int id)
+    {
+        var sql = $"SELECT * FROM level_metadata WHERE LevelID={id}";
```

Comment:
Query strings can also be stored in static const classes. E.g.:
`public static class LevelMetadataQueries{ public static string GetLevelMetadataById => "SELECT * FROM level_metadata WHERE LevelID=@Id" }` notice the @Id part this can be passed as parameter to `QuerySingleAsync` method using dynamic objects or my recommendation Dappers `DynamicParameters()`

Reply:
For now this approach might be fine but as queries get more complex it make sense to move them.